modules/nw-egress-ips-object.adoc

```diff
@@ -2,11 +2,17 @@
 //
 // * networking/ovn_kubernetes_network_provider/assigning-egress-ips-ovn.adoc
 
+:_mod-docs-content-type: REFERENCE
 [id="nw-egress-ips-object_{context}"]
 = EgressIP object
 
 The following YAML describes the API for the `EgressIP` object. The scope of the object is cluster-wide; it is not created in a namespace.
 
+[IMPORTANT]
+====
+EgressIP selected pods cannot serve as backends for services with `externalTrafficPolicy` set to `Local`. If you try this configuration, service ingress traffic that targets the pods gets incorrectly rerouted to the egress node that hosts the EgressIP. This situation negatively impacts the handling of incoming service traffic and causes connections to drop. This leads to unavailable and non-functional services.
+====
```

Comment:
I might bump this to a Warning. See the [RH Supplementary Style Guide](https://redhat-documentation.github.io/supplementary-style-guide/#admonitions):

`Information about potential system damage, data loss, or a support-related issue if the user disregards this admonition.`